examples/observe_usage/main.rs

```diff
@@ -0,0 +1,37 @@
+extern crate observer;
+extern crate uuid;
+mod policy;
+mod user_id;
+
+use crate::{policy::Policy, user_id::UserId};
+use chrono::{offset::TimeZone, Utc};
+use observer::queue::QueueEnum;
+use observer::{context::Context, observer::observe, AResult};
+
+// run this one to see the example ``
+fn main() {
+    let ctx = Context::new(String::from("example_context"), QueueEnum::Kafka);
```

Comment:
Kafka this should be in another crate, observer_kafka maybe, we do not want compile time dependency. Or we can  use feature flags.

Remove the first argument, I am not sure what it will be, its going to a constant string, lets add parameters when we have a use case instead of gratuitously like this, lets focus on absolutely minimal that is needed for things to work, instead of thinking of future cases. In future we can add those things back when we have clarity/consensus.

Reply:
This means second argument is going to a boxed trait implementation.